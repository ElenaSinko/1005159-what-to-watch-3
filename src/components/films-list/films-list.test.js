import * as React from "react";
import * as renderer from "react-test-renderer";
import FilmsList from "./films-list";
import {BrowserRouter} from "react-router-dom";

const smallMovieCards = [
  {
    title: `Test`,
    img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  }, {
    title: `Test`,
    img: `img/bohemian-rhapsody.jpg`,
  }
];

it(`Render FilmsList`, () => {
  const tree = renderer
    .create(<BrowserRouter>
      <FilmsList
        smallMovieCards={smallMovieCards}
      />
    </BrowserRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
