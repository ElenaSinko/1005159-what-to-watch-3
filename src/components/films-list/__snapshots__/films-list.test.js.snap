// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FilmsList 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <a
      href="/film/undefined"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <video
          className="player__video"
          height="175"
          muted={true}
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
          style={
            Object {
              "color": "white",
            }
          }
        />
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <a
      href="/film/undefined"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <video
          className="player__video"
          height="175"
          muted={true}
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
          style={
            Object {
              "color": "white",
            }
          }
        />
      </h3>
    </a>
  </article>
</div>
`;
